### DISCUSSION TITLE
urql pronunciation

### DISCUSSION THREAD

**Discussion by o-az** [urql-graphql/urql#3591] [Q&A] [answered]
Is `urql` pronounced like **\\sequel\\** or **\\ess-kew-ell\\**?

**Comment by JoviDeCroock** [comment, marked as answer]
It's pronounced like erkel, similar to [Steve Urkel](https://en.wikipedia.org/wiki/Steve_Urkel)

**Comment by solrik** [comment]
Er-kull

[notes](https://clumsy-humor-894.notion.site/Client-side-GraphQL-with-React-4248372d51604858aaf9eeb9127b6433)
